src/main/resources/bootstrap.yml

```diff
@@ -0,0 +1,5 @@
+
+spring:
```

Comment:
we can set it in appliation.properties as `spring.application.name=bar`

Reply:
I followed spring sleuth documentation so added that. Sure will make this change.